### ISSUE TITLE
Compatible with iosevka 19?

### ISSUE THREAD

**Issue by DamienCassou** [protesilaos/iosevka-comfy#4]
I'm trying to build iosevka-comfy on top of iosevka 19 and I get a build error:

```
Error: Error: Composite buildPlans.iosevka-comfy cannot be resolved: VXDD,2.
    at Composite.decompose (file:///build/source/font-src/support/variant-data.mjs:173:19)
    at Composite.resolve (file:///build/source/font-src/support/variant-data.mjs:188:39)
    at Composite.resolve (file:///build/source/font-src/support/variant-data.mjs:186:31)
    at Composite.resolve (file:///build/source/font-src/support/variant-data.mjs:186:31)
    at Module.apply (file:///build/source/font-src/support/variant-data.mjs:17:17)
    at createParaImpl (file:///build/source/font-src/index.mjs:41:15)
    at reinit (file:///build/source/font-src/index.mjs:53:16)
    at main (file:///build/source/font-src/index.mjs:98:67)
    at ChildProcess.<anonymous> (/build/source/node_modules/verda/lib/actions/nodejs/command.js:53:31)
    at ChildProcess.emit (node:events:513:28)
    at emit (node:internal/child_process:937:14)
    at process.processTicksAndRejections (node:internal/process/task_queues:83:21)
```

**Comment by lunik1** [comment]
It seems like upstream doesn't necessarily follow the latest Iosevka release. Maybe `iosevka-comfy` should clone its own copy based on the pin given in `iosevka-comfy`'s README (currently v16.3.4, commit `ad1e247a`)?

**Comment by protesilaos** [comment]
Hello @DamienCassou and @lunik1!  The plan is to make Iosevka Comfy build on top of the latest release of Iosevka.  This will need some more time though.  The upstream project makes regular releases that have breaking changes.  I need to try them out before publishing anything.  Testing is time-consuming due to how long it takes to rebuild font files.  Maybe I can do this on Thursday.

**Comment by protesilaos** [comment]
@DamienCassou In your backtrace the problem is with character `VXDD` which has been changed to `cv99`.  Maybe it will work if you change this in the build file?

```
VXDD = 2    # ijäöü square diacretics
```

Again though, there are more changes and I need to be sure we don't have stylistic regressions.  It happened in the past where characters basically swapped places and I got the wrong styling for them.
